nmdc_runtime/site/translation/neon_translator.py

```diff
@@ -0,0 +1,819 @@
+import re
+import math
+import sqlite3
+
+import pandas as pd
+
+from nmdc_schema import nmdc
+from nmdc_runtime.site.translation.translator import Translator
+from nmdc_runtime.site.util import get_basename
+
+
+class NeonDataTranslator(Translator):
+    def __init__(self, mms_data: dict, sls_data: dict, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+
+        self.conn = sqlite3.connect("neon.db")
+
+        neon_mms_data_tables = (
+            "mms_metagenomeDnaExtraction",
+            "mms_metagenomeSequencing",
+        )
+
+        neon_sls_data_tables = (
```

Comment:
These are getting defined at multiple places in the code - here and twice in repository.py. It would be better to define these kinds of things (keys, lists of tables to translate, etc.) once for a given data source